sdk/include/psdk/winuser.h

```diff
@@ -1825,6 +1825,19 @@ extern "C" {
 #define WM_MOUSEHOVER               0x02A1
 #define WM_MOUSELEAVE               0x02A3
 
+#define WTS_CONSOLE_CONNECT 0x1
```

Comment:
We have to check whether this is Vista+; if so, this has to be surrounded by suitable #ifdefs.

Reply:
`#if (WINVER >= 0x0600)` added.

Reply:
To be compatible with MS it would be `#if(_WIN32_WINNT >= 0x0501)`